api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -78,4 +89,30 @@ public Void handle(ExecutionContext context) {
     */
     @Nonbinding
     String fallbackMethod() default "";
+
+    /**
+     * The list of exception types which should trigger Fallback
+     * <p>
+     * Note that if a method throws a {@link Throwable} which is not an {@link Error} or {@link Exception}, 
+     * non-portable behavior results.
+     *
+     * @return the exception types which should trigger Fallback
+     */
+    @Nonbinding
+    Class<? extends Throwable>[] applyOn() default {Throwable.class};
```

Comment:
May I ask to add a sentence on what semantics an empty list has? I assume it means no white-list is used that any exception triggers the fallback.

Reply:
Well, the default is "any exception [Throwable] triggers fallback", so presumably if the user specifies an empty list, they want no fallback. Sounds crazy/useless, but perhaps there might be some usecases for overriding a non-empty list specified in an annotation with an empty list specified in a config file.

Reply:
But I agree this corner case should be explicitly specified for each annotation/attribute.

Reply:
> But I agree this corner case should be explicitly specified for each annotation/attribute.

That is what I was after. Ideally reading the javadoc should spare people reading the spec.